spec/features/user_edits_profile_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+feature 'user edits profile', %Q{
+  As an authenticated user, I want to edit my profile, so that I can distinguish myself from other users.
+  Acceptance Criteria:
+  * [X] - I can update my name.
+  * [X] - I can update my location.
+  * [X] - I can update my age.
+  * [X] - I can update my website.
+  } do
+
+  let(:user) do
+    FactoryGirl.create(:user)
+  end
+
+  scenario 'user edits optional information' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.